content.go

```diff
@@ -98,3 +100,22 @@ func Resolve(ctx context.Context, target Target, ref string, opts ResolveOptions
 
 	return selectPlatform(ctx, target, desc, opts.TargetPlatform)
 }
+
```

Comment:
How about moving these two methods to a separate package and file, like `descriptor/descriptor.go`?

Reply:
Done. Used the package `oras` as all files of the same level belong to `oras`.

Reply:
Created package descriptor.